### ISSUE TITLE
Privoxy not Using VPN Connection, Exposes Private IP

### ISSUE THREAD

**Issue by Ryonez** [binhex/arch-delugevpn#317]
So, I decided today to set things up so I can use Privoxy on my desktop browser.

I set up a reverse proxy to the port with my swag container:
![image](https://user-images.githubusercontent.com/16157130/179099892-9edce1af-3fd8-4575-8130-403d9340a2bc.png)

And added it to the SwitchyOmega extension on Vivaldi:
![image](https://user-images.githubusercontent.com/16157130/179100032-9f762a9c-7334-4547-bc39-396feb7adf0a.png)

However, while it all seems to work, the proxied browser returns my home IP rather than the VPN.

![image](https://user-images.githubusercontent.com/16157130/179100697-9f3c1ab7-2208-49fb-9f7e-5eafe6c29137.png)

Deluge seems to show a different ip:
![image](https://user-images.githubusercontent.com/16157130/179100812-74ec5f05-ca75-45e1-ac85-f73e728f5024.png)

This seems broken to me, and I'm concerned that this means that my *arr containers that have been configured with proxy settings are exposing the ip as well now.

**Comment by binhex** [comment]
im quite confused by your setup, so you are reverse proxying a forwarding (privoxy) proxy?! this doesnt sound like a good plan to me. if you want to use the proxy internally then simply set your browser to use the internal ip and port assigned to privoxy, i would not advise exposing privoxy to the internet!.

**Comment by Ryonez** [comment]
> reverse proxying a forwarding (privoxy) proxy?!
Yes I am, I allows me to form a signed secure connection via my domain.

> i would not advise exposing privoxy to the internet!.
Indeed that would be a bad idea. This service has access control to only allow my local networks though. The location block in the ngnix shows my LAN and VPN ranges.

So I do this for two reasons.
The first I already mentioned, serving it through swag allows my to make secure connection with my domain, while restricting sensitive services.
The second is I stick most of my services on their own docker network. I do not expose the ports where I can, and serve most things secured through swag. No sending password in the clear (bar pihole, that's complicated), and no messing with self signed certificates.
